plotly/tests/test_core/test_tools/test_TraceFactory.py

```diff
@@ -0,0 +1,89 @@
+from unittest import TestCase
+from plotly.graph_objs import graph_objs, Line
+from plotly.exceptions import PlotlyError
+
+import plotly.tools as tls
+import math
+from nose.tools import raises
```

Comment:
:cow2: also minor, but good to order imports as:

```
# standard lib "import <whatever>"
# standard lib "from <whereever> import <whatever>"

# 3rd party lib "import <whatever>"
# 3rd party lib "from <wherever> import <whatever>"

# this module "import <whatever>"
# this module "from <whereever> import <whatever>"
```

And alphabetize each section and each list of imports from a place.

Here are some examples for how we organize this in our backend code:

https://github.com/plotly/dev-docs/blob/master/reference/styleguides/python.md#imports

for this, just forget about the django schtuff in there :)

so:

```
import math
from unittest import TestCase

from nose.tools import raises

import plotly.tools as tls
from plotly.exceptions import PlotlyError
from plotly.graph_objs import Line, graph_objs
```